tests/conftest.py

```diff
@@ -44,6 +43,99 @@ def teardown():
     request.addfinalizer(teardown)
 
 
+def _add_framework(request, app, slug, status, framework, framework_agreement_details=None):
+    if framework_agreement_details:
+        framework_agreement_details = {'frameworkAgreementVersion': 'v1.0'}
+
+    with app.app_context():
+        framework = Framework(
+            slug=slug,
+            name=slug,
+            framework=framework,
+            framework_agreement_details=framework_agreement_details,
+            status=status,
+            clarification_questions_open=True if status == 'open' else False
+        )
+        db.session.add(framework)
+        db.session.commit()
+
+        def teardown():
+            with app.app_context():
+                framework = Framework.query.filter(Framework.slug == slug).first()
+                Framework.query.filter(Framework.id == framework.id).delete()
+                db.session.commit()
+
+    request.addfinalizer(teardown)
+
+
+def _framework(request, app, slug, status, framework, framework_agreement_details=None):
```

Comment:
Although I like the consistency of this with the other function names, I can't help but feel the function name should be more descriptive, given that `_framework` tells very little about what this function is doing. Not sure though, what do you think?